app/src/main/java/com/duckduckgo/app/pixels/EnqueuedPixelWorker.kt

```diff
@@ -44,7 +44,8 @@ class EnqueuedPixelWorker @Inject constructor(
     private val workManager: WorkManager,
     private val pixel: Provider<Pixel>,
     private val unsentForgetAllPixelStore: UnsentForgetAllPixelStore,
-    private val webViewVersionProvider: WebViewVersionProvider
+    private val webViewVersionProvider: WebViewVersionProvider,
+    private val APP_RESTART_CAUSED_BY_FIRE_GRACE_PERIOD: Long = 10_000L
```

Comment:
This constant should not be in the constructor. It is not needed there. I would revert this change and leave the constant where it was before